src/Compatibility/Core/src/Windows/FlyoutPageRenderer.cs

```diff
@@ -303,8 +304,8 @@ void UpdateBounds()
 			global::Windows.Foundation.Size masterSize = Control.FlyoutSize;
 			global::Windows.Foundation.Size detailSize = Control.DetailSize;
 
-			Element.FlyoutBounds = new Rectangle(0, 0, masterSize.Width, masterSize.Height);
-			Element.DetailBounds = new Rectangle(0, 0, detailSize.Width, detailSize.Height);
+			FlyoutPageController.FlyoutBounds = new Rectangle(0, 0, masterSize.Width, masterSize.Height);
```

Comment:
These changes are just related to changing the `IFlyoutPageController` properties so they use explicit interface opposed to `Browsable.None`